src/views/canvas/CanvasSeqBlock.js

```diff
@@ -175,29 +174,30 @@ const View = boneView.extend({
   },
 
   render: function() {
+    const width = this.getPlannedElWidth();
+    const height = this.getPlannedElHeight();
+
     if (this.g.config.get("shouldRenderSeqBlockAsSvg") === true) {
       this.el.setAttributeNS("http://www.w3.org/2000/svg", 'height', this.getPlannedElHeight());
-      this.el.setAttributeNS("http://www.w3.org/2000/svg", 'width', this.getPlannedElWidth());
-      this.el.style.width = `${this.getPlannedElWidth()}px`;
-      this.el.style.height = `${this.getPlannedElHeight()}px`;
+      this.el.setAttributeNS("http://www.w3.org/2000/svg", 'width', width);
+      this.el.style.width = `${width}px`;
+      this.el.style.height = `${height}px`;
     } else {
       this.el.adjustSize({
-        height: this.getPlannedElHeight(),
-        width: this.getPlannedElWidth(),
+        height,
+        width,
       })
     }
 
     if (this.g.config.get("shouldRenderSeqBlockAsSvg") === true) {
-      const width = this.getPlannedElWidth();
-      const height = this.getPlannedElHeight();
       this.ctx = new C2S(width, height)
     }
 
     this._setColor();
 
     this.seqDrawer = new CanvasSeqDrawer( this.g,this.ctx,this.model, {
-      width: this.el.width,
-      height: this.el.height,
+      width,
+      height,
```

Comment:
Fun fact: This actually fixes a bug in our virtualization logic where we ended up rendering 4x times the amount of blocks than the original in some systems (like my Mac).
I found this by using the render stats module 🙈, because it reported more than what's actually visible on the screen.

`this.el.width` is twice as big as `width`.
The latter is the actual pixel size, and what's actually used for virtualization, where as the former became twice of that because of [`devicePixelRatio`](https://developer.mozilla.org/en-US/docs/Web/API/Window/devicePixelRatio) being set to 2.

Reply:
nice